clients/cloud/clojure/src/io/confluent/examples/clients/clj/consumer.clj

```diff
@@ -11,28 +11,28 @@
 (defn- build-properties [config-fname]
   (with-open [config (jio/reader config-fname)]
     (doto (Properties.)
-      (.putAll {ConsumerConfig/GROUP_ID_CONFIG, "clojure_example_group"
-                ConsumerConfig/KEY_DESERIALIZER_CLASS_CONFIG "org.apache.kafka.common.serialization.StringDeserializer"
+      (.putAll {ConsumerConfig/GROUP_ID_CONFIG,                "clojure_example_group"
```

Comment:
```suggestion
      (.putAll {ConsumerConfig/GROUP_ID_CONFIG                "clojure_example_group"
```
I don't think we need the comma here. Commas are considered whitespace in Clojure, so for maps defined on a single line (e.g. `{:foo foo-val, :bar bar-val}`), it helps readability to add them. For multi-lines one, like here, they're redundant.

Reply:
Okay -- I'm using emacs and [cider](https://github.com/clojure-emacs/cider) which has a nice [format function. ](https://github.com/clojure-emacs/cider/blob/master/cider-format.el).